gear/gear/auth_utils.py

```diff
@@ -1,10 +1,30 @@
+import logging
+import os
 import secrets
 from typing import Optional
 
 from hailtop.auth import session_id_encode_to_str
 
 from .database import Database
 
+MAX_AGE_SECS = None
+
+
+def max_age():
+    global MAX_AGE_SECS
+    if MAX_AGE_SECS is None:
+        if (s := os.getenv('SESSION_MAX_AGE_SECS')) is not None:
+            try:
+                MAX_AGE_SECS = int(s)
+            except ValueError:
+                logging.exception("Unable to interpret SESSION_MAX_AGE_SECS as an integer.")
+
+    if MAX_AGE_SECS is None:
+        # Default value: 1800 seconds (30 minutes)
+        MAX_AGE_SECS = 1800
```

Comment:
I think the ideal default would be more like "whatever it is today", so maybe more like 30 days?